tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -0,0 +1,130 @@
+# LibreOffice tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+# Case 1503827 - LibreOffice: Launch application components from system menu
+
+# open desktop mainmenu and click office
+sub open_mainmenu() {
+    my $self = shift;
+
+    wait_still_screen;
+    send_key "alt-f1";
+    assert_screen 'test-desktop_mainmenu-1', 20;
+    assert_and_click 'mainmenu-office';
+    assert_screen 'mainmenu-office-components', 20;
+}
+
+# enter 'Activities overview'
+sub open_overview() {
+    my $self = shift;
+
+    wait_still_screen;
+    send_key "super";
+    assert_screen 'tracker-mainmenu-launched';
+}
+
+sub run() {
+    my $self = shift;
+
+    # launch components from mainmenu
+    $self->open_mainmenu();
+    assert_and_click 'mainmenu-office-lo';    #open lo
+    assert_screen 'welcome-to-libreoffice';
+    send_key "ctrl-q";                        #close lo
+
+    $self->open_mainmenu();
+    assert_and_click 'mainmenu-office-base';    #open base
+    assert_screen 'oobase-select-database', 45;
+    send_key "ret";
+    sleep 3;
```

Comment:
Sleep is not necessary, the assert_screen will take care of the delay you want here